File: Unit 1/Tower Of Hanoi/TowerOfHanoi.java
class TowerOfHanoi
{
    static void Hanoi(int n, char A, char B, char C)
    // A: Start , B: Destination, C: Auxiliary
    {
        if (n == 1)
        {
            System.out.println("Move disk from "+ A + " to " + B);
        }

        else 
        {
            Hanoi(n-1, A, C, B);
            System.out.println("Move disk from "+ A + " to " + B);
            Hanoi(n-1, C, B, A);
        }
    }

    public static void main(String[] args)
    {
        Hanoi(4, 'A', 'B', 'C');
    }
}

File: Unit 1/Tower Of Hanoi/TowerOfHanoi.py
def Hanoi(n:int, A:str, B:str, C:str):
    # A: Start , B: Destination, C: Auxiliary
    if n == 1:
        print(f'Move disk form {A} to {B}')
    else:
        Hanoi(n-1, A, C, B)
        print(f'Move disk from {A} to {B}')
        Hanoi(n-1, C, B, A)

Hanoi(3, 'A', 'B', 'C')